src/components/RnR/RnR_RatingBreakdown.jsx

```diff
@@ -1,46 +1,16 @@
-import React, { Component } from 'react';
+import React from 'react';
 import StarRating from './RnR_StarRating';
+import StarBreakdown from './RnR_StarBreakdown';
```

Comment:
I think this is where your linting error is coming from, since your default export was named ConnectedStarBreakdown,it expects you to import it with that same name